drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -36,7 +36,12 @@ class Forms::FormDefinition < Types::BaseObject
     # we might need to apply filter. Probably the filtering should get moved to it's own
     # class down the road
     def definition
-      eval_items([object.definition])[0]
+      Hmis::Form::DefinitionItemGraphqlAdapter.perform(
```

Comment:
I think the PR is missing a file @ttoomey? I don't see this class

Reply:
Oh, looks like a rename of. DefinitionItemFilter

Reply:
Oops, yes I forgot to update this after I renamed it. Thanks!